pkgs/servers/waterfall/config.yml

```
@@ -0,0 +1,52 @@
+connection_throttle: 4000
+connection_throttle_limit: 3
+timeout: 30000
+stats: 69135692-e718-4282-a863-3231fb7fef22
+permissions:
+  default:
+    - bungeecord.command.server
+    - bungeecord.command.list
+  admin:
+    - bungeecord.command.alert
+    - bungeecord.command.end
+    - bungeecord.command.ip
+    - bungeecord.command.reload
+    - bungeecord.command.kick
+network_compression_threshold: 256
+servers:
+  lobby:
+    motd: "&1Just another Waterfall - Forced Host"
+    address: localhost:25565
+    restricted: false
+player_limit: -1
+prevent_proxy_connections: false
+server_connect_timeout: 5000
+enforce_secure_profile: false
+remote_ping_timeout: 5000
+forge_support: true
+remote_ping_cache: -1
+log_commands: false
+log_pings: true
+ip_forward: false
+disabled_commands:
+  - disabledcommandhere
+groups:
+  md_5:
+    - admin
+listeners:
+  - host: 0.0.0.0:25577
+    query_port: 25577
+    motd: "&1Another Bungee server"
```

A little bit of customization
```suggestion
    motd: "&1Another Mineflake server"
```